src/components/user/LogoutButton.js

```diff
@@ -8,9 +8,9 @@ const LogoutButton = () => {
   const logoutRequest = useDispatch(logout);
 
   return (
-    <button onClick={logoutRequest} type="button">
+    <a style={{ cursor: 'pointer' }} onClick={logoutRequest}>
```

Comment:
I think it's better to create a class named `cursor-pointer` or something like that and use it here